src/style/app.module.css

```diff
@@ -308,13 +197,6 @@
   border: 0;
 }
 
-.inputField {
-  margin-top: 10px;
-  margin-bottom: 10px;
```

Comment:
Why was this removed? Won't it have an impact on other layouts?

Reply:
it's already exists on line 112